datashader/transfer_functions.py

```diff
@@ -245,7 +247,7 @@ def set_background(img, color=None):
     if color is None:
         return img
     background = np.uint8(rgb(color) + (255,)).view('uint32')[0]
-    data = source(img.data, background)
+    data = over(img.data, background)
```

Comment:
Why did you change this? This doesn't set the background, it overlays an image over a background of that color. For images that use alpha to indicate magnitude (output of `colorize`), this will make all set pixels have full alpha. I'm unsure if this is desired here.